src/transforms.h

```diff
@@ -122,7 +141,7 @@ class ScTransformer : public scBaseVisitor
         antlrcpp::Any result = ctx->expression()->accept(this);
         std::shared_ptr<Expression> expression = result.as<std::shared_ptr<Expression>>();
 
-        auto node = std::make_shared<InitializerNode<LatticeType>>(ctx->type()->getText(), ctx->ID()->getText(), expression);
+        auto node = std::make_shared<AssignmentNode<LatticeType>>(ctx->ID()->getText(), expression);
```

Comment:
type information is lost, might as well just remove initialization from the grammar since the type information is ignored

Reply:
Type information is redundant to this analysis. Initialization wont be shouldnt be removed from syntax as it gets us slightly closer to proper C syntax